ui/apps/platform/src/services/ComplianceEnhancedService.ts

```diff
@@ -255,6 +252,16 @@ export function getAllClustersCombinedStats(
         });
 }
 
+export function getAllClustersCombinedStatsCount(): Promise<number> {
+    return axios
+        .get<{
+            count: number;
```

Comment:
Comment about something that is to developer discretion.

During the past year or so, I have tended to write TypeScript equivalent to backend response types:
https://github.com/stackrox/stackrox/blob/master/proto/api/v2/compliance_results_service.proto#L126-L128

```ts
type CountComplianceScanResults = {
    count: number; // int32
};
```

and then

```ts
        .get<CountComplianceScanResults>(…)
```